frontend/app/src/store/history/actions.ts

```diff
@@ -391,6 +422,23 @@ export const actions: ActionTree<HistoryState, RotkehlchenState> = {
     }
   },
 
+  async [HistoryActions.PURGE_MOVEMENTS](
+    { commit },
+    location: SupportedExchange
+  ): Promise<void> {
+    commit(HistoryMutations.ADD_MOVEMENT_LOCATION_TO_PURGE, location);
+    const params = {
+      location,
+      limit: 1,
+      offset: 0,
+      ascending: false,
+      orderByAttribute: 'time'
```

Comment:
Question: Isn't the sort field preserved when navigating between tabs? Is the sort by time here a problem?

Reply:
These defaults are only for when the parameters are missing from the action.

The pagination event from the table will override these values.
Essentially these defaults are used only for refresh, where the result is ignored by the frontend, since the frontend is now using the `only_cache: true` (db) as a single source of truth.